File: solutions/javascript/darts/1/darts.js
//
// This is only a SKELETON file for the 'Darts' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const score = (x, y) => {
  var distance = Math.sqrt(x*x + y*y)
  if (distance > 10) return 0;
  if (distance > 5) return 1;
  if (distance > 1) return 5;
  return 10;
  //throw new Error('Remove this statement and implement this function');
};

File: solutions/csharp/darts/1/Darts.cs
using System;

public static class Darts
{
    public static int Score(double x, double y)
    {
        if (Math.Sqrt(x*x+y*y) > 10) return 0;
        if (Math.Sqrt(x*x+y*y) > 5) return 1;
        if (Math.Sqrt(x*x+y*y) > 1) return 5;
        return 10;
    }
}
